foodsaving/groups/tasks.py

```diff
@@ -0,0 +1,12 @@
+from huey import crontab
+from huey.contrib.djhuey import db_periodic_task
+
+from foodsaving.groups import stats
+from foodsaving.groups.models import Group
+
+
+@db_periodic_task(crontab(minute='*'))
```

Comment:
Oh, I left it to report every minute... was just going to make sure it is all working, then push a change to reduce it to, perhaps, once per day.